spec/02-integration/06-invalidations/02-core_entities_invalidations_spec.lua

```diff
@@ -24,6 +24,7 @@ describe("core entities are invalidated with db: " .. kong_conf.database, functi
     local kong_dao_factory = require "kong.dao.factory"
     dao = assert(kong_dao_factory.new(kong_conf))
     dao:truncate_tables()
+    helpers.run_migrations(dao)
```

Comment:
Not sure what the value is in running the migrations after the truncate? It will throw a lot of errors on the testing DB server (truncating non-existing tables) - sure, we don't care that much. But running them before allows us to make sure the DB is up-to-date before emptying it (it could very will be up-to-date, but populated with leftover fixtures). Or else I am missing something?